docs(azure): Add pointer to how to add env variables (#341)
unjs/nitro: docs/deploy/providers/azure.md

```

That's it! Now Azure Static Web Apps will automatically deploy your Nitro-powered application on push.

> In case you are using server runtime configuration, you likely want to configure the corresponding [environment variables on Azure](https://docs.microsoft.com/en-us/azure/static-web-apps/application-settings).

Reviewer: ```suggestion

If you are using runtimeConfig, you will likely want to configure the corresponding [environment variables on Azure](https://docs.microsoft.com/en-us/azure/static-web-apps/application-settings).
```
